themes_cern/indico_themes_cern/templates/council.html

```diff
@@ -2,3 +2,28 @@
 
 {# Hide location #}
 {% set show_siblings_location = false %}
+
+{# Customize "day header" to add time information #}
+{% block day_header %}
+    <div class="day-header" style="width: 100%;">
+        <div class="day-title" data-anchor="{{ anchor }}">
+            {{ item.start_dt | format_date(format='EEEE, d MMMM', timezone=timezone) }}
+            at
+            {{ item.start_dt | format_time(format='short') }}
+        </div>
+        {% if days|length > 1 %}
+            <a href="" class="js-go-to-day icon-calendar arrow js-dropdown" data-toggle="dropdown"></a>
+
+            <ul class="dropdown days-dropdown">
+                {% for day, _ in days %}
+                    <li>
+                        <a href="#day-{{ day.isoformat() }}">
+                            {{ day | format_date(format='EEE, d MMM', timezone=timezone) }}
+                        </a>
+                        <span>
```

Comment:
Why is it here?

Reply:
Thanks! I forgot to delete that, clearly.